lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1927,10 +1930,10 @@ sub investigate ($self, %args) {
     return \%inv;
 }
 
-sub packages_diff ($self, $prev, $ignore) {
-    my $current_file = path($self->result_dir, 'worker_packages.txt');
+sub packages_diff ($self, $prev, $ignore, $filename) {
```

Comment:
```suggestion
sub packages_diff ($self, $prev, $ignore, $filename, $fallback = 'Diff of packages not available') {
```

And then replace all the `return` statements not returning anything with `return $fallback`. This way we can avoid the duplication of the fallback string and the not so nice wrapping.